kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/metrics/MetricsCollector.java

```diff
@@ -145,51 +158,6 @@ public String getNamespaceName() {
         return namespaceName;
     }
 
-    /**
```

Comment:
I don't understand why you take these bits away.  The STs will still need to scrape metrics.

Reply:
ah okay!